src/constructs/aws/StaticWebsite.ts

```diff
@@ -55,4 +57,21 @@ export class StaticWebsite extends StaticWebsiteAbstract {
             code: cloudfront.FunctionCode.fromInline(code),
         });
     }
+    /**
+     * Overrides the default `getBucketProps` from the abstract class
+     *
+     * @returns bucketProps
```

Comment:
We're not using JS Doc to provide return type of function, only return Type from Typescript. Can you remove this line ?